src/server.h

```diff
@@ -2595,10 +2594,16 @@ int serverCommunicateSystemd(const char *sd_notify_msg);
 void serverSetCpuAffinity(const char *cpulist);
 void dictVanillaFree(dict *d, void *val);
 
+/* ERROR STATS constants */
+#define ERROR_STATS_LIMIT 128 /* Once the errors RAX reaches this limit, instead of tracking custom \
```

Comment:
it is odd the comment had a `\`

Reply:
When we let clang-format fix this, it's not always a nice result. I think it's better to put the comment on its own line, for all these.

Idea:

```c
/* --- ERROR STATS constants --- */

/* Once the errors RAX reaches this limit, instead of tracking custom
 * errors (e.g. LUA), we track the error under the prefix below. */
#define ERROR_STATS_LIMIT 128

#define ERRORSTATS_OVERFLOW_ERR "ERRORSTATS_OVERFLOW"

/* --- afterErrorReply flags --- */

/* Indicating that we should not update error stats after sending error reply. */
#define ERR_REPLY_FLAG_NO_STATS_UPDATE (1ULL<<0)

/* Indicates the error message is custom (e.g. from LUA). */
#define ERR_REPLY_FLAG_CUSTOM (1ULL<<1)
```

Reply:
Btw, I see now that the prefix for one macro is `ERROR_STATS` and the other one has `ERRORSTATS` prefix. Use the same prefix for both maybe?